tests/test/builtin_impls/unsize.rs

```diff
@@ -68,9 +68,7 @@ fn dyn_to_dyn_unsizing() {
             "Unique; substitution [], lifetime constraints [InEnvironment { environment: Env([]), goal: '!1_0: '!1_0 }]"
         }
 
-        // Different order of traits in target and source
-        // FIXME: this doesn't work because trait object unification
-        // respects where clause order, which it shouldn't
+        // Different order of traits in target and sourcex
```

Comment:
Sorry, I didn't mention it, but I think there is maybe a similar comment elsewhere in the file, the other test you edited...?

Reply:
it just says, "See above", which I think is still relevant, no ?